syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -720,6 +727,18 @@ def chunk(self, shares, *args, **kwargs):
 
         return results
 
+    @overloaded.method
+    def mean(self, shares, **kwargs):
+        result = {}
+        m = 0
+        for worker, share in shares.items():
+            sum_value = share.sum(**kwargs)
+            if m == 0:
+                m += share.numel() // sum_value.numel()
```

Comment:
We can maybe having the  `m` and `sum_value` initialization to just happen just once, why include it in the loop?

```suggestion
        sum_value = share.sum(**kwargs)
        m = next(iter(shares.values())).numel() // sum_value.numel()
        for worker, share in shares.items():
```